partiql-lang/src/main/kotlin/org/partiql/lang/graph/Graph.kt

```diff
@@ -66,11 +66,11 @@ interface Graph {
  *  based on labels at them.
  */
 sealed class LabelSpec {
-    /** A graph element matches when one of its labels is [name].*/
-    data class OneOf(val name: String) : LabelSpec()
+    /** A graph element (node or edge) matches when its label set contains [name]. */
+    data class Name(val name: String) : LabelSpec()
 
-    /** A graph element always matches (even when it is not labeled at all).*/
-    object Whatever : LabelSpec()
+    /** A graph element matches as long as it has a label. */
+    object Wildcard : LabelSpec()
```

Comment:
While this is *somewhat* considered experimental (under graph, but not annotated with an experimental annotation), if anyone has used this API -- these changes would be considered breaking. IMO, not worthy of deprecation, however, a notice in the CHANGELOG could be a good idea (especially so that we internally remember to release this as a major version bump).

Reply:
Ok, I'll add better communication about this.